HOWTO.md

```diff
@@ -523,10 +524,41 @@ Please mind how the configuration file must be specifically named (e.g. `-` VS
    `/etc/fdo/manufacturing-serverr.conf.d/`. The paths will be checked in that
    same order.
 
-3. Execute `fdo-manufacturing-server` or run it as a service, see sample
-   file in
+3. Execute `fdo-manufacturing-server` using below command:
+   ```
+   sudo MANUFACTURING_SERVER_CONF=/usr/share/fdo/manufacturing_server.yml LOG_LEVEL=trace ./target/debug/fdo-manufacturing-server
+   ```
+   or run it as a service, see sample file in
    [examples/systemd](https://github.com/fedora-iot/fido-device-onboard-rs/blob/main/examples/systemd/fdo-manufacturing-server.service).
 
+### Manufacturing Client
+
+1. Once you have generated required keys/certificates for the Manufacturing Server and manufacturing-server is running
+   (Refer above section: Manufacturing Server) you can run manufacturing-client. Use case such as: if you need to check 
```

Comment:
Please add links to the sections that you are referring to.

Use the standard names for the binaries when referring to them `fdo-manufacturing-server` and `fdo-manufacturing client`.

In this guide we also assume that the user has a general knowledge of the protocol, so there is no need to say that the Manufacturing server must be up and running, but it would be a nice addition. You can add a general disclaimer in the 'How to run the clients' section that the servers related to the specific client (`fdo-client-linuxapp` or `fdo-manufacturing-client`) must be up and running for the clients to work.